spec/runner_spec.rb

```diff
@@ -12,7 +12,7 @@
 
       Cert::Runner.new.launch
       expect(ENV["CER_CERTIFICATE_ID"]).to eq("cert_id")
-      expect(ENV["CER_FILE_PATH"]).to eq("./cert_id.cer")
+      expect(ENV["CER_FILE_PATH"]).to eq("#{Dir.pwd}/cert_id.cer")
```

Comment:
In order for cert to work with both relative and absolute paths, I had to use File.expand to convert the relative path to an absolute (The same way Gym does).

This test fails unless I change it to be the absolute path. 

I have just noticed a weird quirk though with my solution, that when ran under fast lane if you put "./cache" it actually creates a folder called "Cache" in the fastfile folder, is this correct ?